nestkernel/common_synapse_properties.cpp

```diff
@@ -47,13 +49,31 @@ CommonSynapseProperties::~CommonSynapseProperties()
 }
 
 void
-CommonSynapseProperties::get_status( DictionaryDatum& ) const
+CommonSynapseProperties::get_status( DictionaryDatum& d ) const
 {
+  if ( weight_recorders_.size() != 0 )
```

Comment:
For vectors, better use `not w_r_.empty()`. Once this becomes an SC pointer, you can turn it into
```C++
def< long >( d, names::weight_recorder, weight_recorder_ ? weight_recorder->get_gid() : -1 );
```
Note: Please add weight_recorder to `nest::names`.